src/ts/components/subtitleslistbox.ts

```diff
@@ -0,0 +1,55 @@
+import {ListBox} from './listbox';
+import {ListSelectorConfig} from './listselector';
+import {UIInstanceManager} from '../uimanager';
+
+/**
+ * A element that is similar to a select box where the user can select a subtitle
+ */
+export class SubtitlesListBox extends ListBox {
+  constructor(config: ListSelectorConfig = {}) {
+    super(config);
+    this.config = this.mergeConfig(config, {
+      cssClass: 'ui-subtitles-listbox',
+    }, this.config);
+  }
+
+  configure(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
+    super.configure(player, uimanager);
+    let selectCurrentSubtitle = () => {
```

Comment:
This is basically the same code as in the `SubtitleSelectBox`, so the code should be shared via a helper/utility class.

Reply:
Hm what about a `SubtitleSwitchHandler` and pass the `bitmovin.PlayerAPI` and a subclass of `ListSelector` and handle all events where something is to update (Player Events & ListSelector Events) within this class?

Reply:
Sounds good!